Library/Homebrew/test/x11_requirement_spec.rb

```diff
@@ -20,7 +20,7 @@
       expect(subject).not_to eql(other)
     end
 
-    it "returns false if the minimum version differs" do
+    it "returns false if the minimum version differs", :needs_macos do
```

Comment:
Can you instead modify this test so it does something on Linux?

Reply:
I've added a test for Linux.